pkgs/development/python-modules/skyfield/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, certifi, numpy, sgp4, jplephem
+, pandas, ipython, matplotlib, assay
+}:
```

Please apply to the others
```suggestion
}:

```